### PR TITLE
NAb template draft

### PR DESCRIPTION
**Pull request by monicagerber**
Thanks for reviewing the NAb template draft. 

To create this draft, I reviewed:
* [Dubois846 NAb report](https://github.com/FredHutch/Dubois846Analysis/blob/nab-pt-report/NAb-pt-report/NAb-pt-report.pdf)
* [Caskey820 NAb report](https://github.com/FredHutch/Caskey820Analysis/blob/master/NAB/Caskey_NAb_Interim_Report/Caskey820_Blinded_Interim_NAb_PT_Report.pdf)
* McElrath [708](https://github.com/FredHutch/McElrath708Analysis/blob/master/NAb/NAb_PT_Report/NAb_PT_Report.pdf) and [749](https://github.com/FredHutch/McElrath749Analysis/blob/master/NAb/NAb_pt_report.pdf) NAb reports
* [Gallo 477 NAb report](https://github.com/FredHutch/Gallo477Analysis/blob/master/NAb/Interim_analyses/Cohort1/PT_NAb_Gallo477_Cohort1/PT_NAb_Gallo477_Cohort1.pdf)
* HVTN reports package [NAb memo template](https://github.com/FredHutch/hvtnReports/blob/master/inst/memo_templates/memo_nab.Rmd)
* [VISC-Assay-Methods](https://github.com/FredHutch/VISC-Assay-Methods) repo

Some notes:
* I used "net ID_50_ titer" (subscript does not work in Github comments!) consistently to refer to the statistical endpoint. This hasn't been consistent in past reports. We've used a combination of SLV-subtracted ID50 titer, ID50 titers, NAb titers, etc. to refer to the subtracted endpoint. 
* There's a note to check for SLV-subtraction in the lab study plan. 
* HVTN uses "IC50".
* I moved the MB description to the graphical analysis and adopted the language from HTVN reports which I thought was easier to understand.

Can you please complete this review by COB 1/20? Let me know if you need additional time. I may schedule a working meeting for us to review edits like we did with BAMA.

Thank you!

### CODE REVIEW

**Comment by monicagerber** [comment]
Hi @mayerbry, we talked about looking at recent publications so we can decide on the best name for the response magnitude. If you could point us to some links or make a suggestion, that'd be helpful. Thanks in advance!

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (celiaeddy) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:13]: I think it can be kept as one template but we could put all references to [net] in brackets.

File: inst/templates/methods-nab/nab-lab-methods.Rmd
Comment (celiaeddy) [inline inst/templates/methods-nab/nab-lab-methods.Rmd:20]: somewhere in this file we could add a link to CATNAP - which is a reference SRAs can use to add clade and tier information. https://www.hiv.lanl.gov/components/sequence/HIV/neutralization/

**Review by celiaeddy — APPROVED** [review]
thanks for putting this together @monicagerber , I responded/agreed with Alicia's comments and added one of my own. let me know if you think it's better to chat for 30 min to help finalize this PR.

File: inst/templates/methods-nab/nab-lab-methods.Rmd
Comment (asatofh) [reply inst/templates/methods-nab/nab-lab-methods.Rmd:20]: jason is helping with a new workflow to pull clade and tier from linda's nab database into pdata, so hopefully sra's don't have to pull from catnap. marie may have additional information if you're interested.

**Review by mayerbry — APPROVED** [review]
I found two recent examples of clinical work with nab results and they use "ID50 Titer" or similar so I think we should stick with that. For pre-clinical work (and rare clinical work) there are background/control titers like MLV. This is not the standard case, but I think there should be a comment reminder that the titer calculation should be verified and explicitly defined if there is subtraction. The correct notation (ex, net ID50 titer or MLV-subtracted titer) for plots can be established per conversation with lead stat/lab under those circumstances. 


[HVTN405](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1003868)
[Gallo477](https://www.sciencedirect.com/science/article/pii/S0264410X2100685X)

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:10]: I edited this so it says "Check with the lab and lead statistician." 
Not sure where to record background information about using MLV concentrations/deferring to the lab.

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:13]: I added a note at the top:

> If using SLA-MLV-subtracted values, replace "ID~50~ titer" with "net ID~50~ titer" or "MLV-subtracted titer" (use one, be consistent).

We want to avoid two templates, that's more to maintain and will become a source of conflicting updates.

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:38]: Yeah, this is unfortunately a relic of past templates and reports. I can remove "experimental" from this template. In the future, it may be a good idea to state a preference or provide guidelines for using the following: "vaccine groups", "study groups", "experimental groups", "treatment groups", "groups".

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:37]: The template contains text for a comprehensive unblinded report. Rather than add brackets ("[Unblinded report]") next to optional things, I think we should leave it to the SRA to decide on which parts to edit and use, and which parts to remove, based on the SAP and report requirements.

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:61]: Can you provide an example of how you'd like this to be stated?

File: inst/templates/methods-nab/nab-lab-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-lab-methods.Rmd:20]: Add a link to the lab methods template so that SRAs know that this is a resource.

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:49]: This is in Celia's Dubois846 NAb report, linked above.

**Comment by monicagerber** [comment]
> I found two recent examples of clinical work with nab results and they use "ID50 Titer" or similar so I think we should stick with that. For pre-clinical work (and rare clinical work) there are background/control titers like MLV. This is not the standard case, but I think there should be a comment reminder that the titer calculation should be verified and explicitly defined if there is subtraction. The correct notation (ex, net ID50 titer or MLV-subtracted titer) for plots can be established per conversation with lead stat/lab under those circumstances.
> 
> [HVTN405](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1003868) [Gallo477](https://www.sciencedirect.com/science/article/pii/S0264410X2100685X)

Thanks, Bryan. This is helpful. I changed the methods to "ID~50~ titer" and added a note about MLV-subtracted titers to the top.

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:61]: We decided to add this as a separate issue (See #102 )

File: inst/templates/methods-nab/nab-statistical-methods.Rmd
Comment (monicagerber) [reply inst/templates/methods-nab/nab-statistical-methods.Rmd:49]: We decided to swap "durability" for "fold-change" in this template.

**Comment by monicagerber** [comment]
Hi @asatofh, please let me know if you approve these changes. Once you approve, I can merge this PR.

**Pull request merged**
